network/utils.go

```diff
@@ -31,6 +34,24 @@ func Listen(connectionString string) (net.Listener, error) {
 	}
 }
 
+func ListenTCP(connectionString string) (*net.TCPListener, error) {
```

Comment:
https://github.com/cossacklabs/acra/blob/master/network/utils.go#L22 can we use it or extend/replace something of these 2?

Reply:
done